app/src/main/java/org/jellyfin/androidtv/model/itemtypes/Episode.kt

```diff
@@ -7,5 +7,7 @@ data class Episode (
 	//TODO: Chapters: ArrayList<ChapterInfoDto>
 	val communityRating: Double,
 	val name: String,
-	val description: String
+	val description: String,
+	val canResume: Boolean,
```

Comment:
Might be worth splitting the user specific data to a separate class. Something like "Episode" and "UserEpisode". This way we can (in the future) cache episode data in the app when switching users.

Not necessary though

Reply:
Right, we need to very much rethink the whole model though, should we do this here, via Issue or Matrix?

Reply:
Probably in the issue or a new PR

Reply:
Okay, let's move this to a new PR